### PR TITLE
BoF #2 charter issue #6

### PR DESCRIPTION
**Pull request by bledvina**
See issue #109

### CODE REVIEW

File: charter/charter-detecting-unwanted-location-trackers.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ The WG is expected to:
 
 The WG will not standardize an end-to-end platform-based unwanted tracking detection system or define requirements for interactions between accessory manufacturers and law enforcement. Phones, tablets, watches, and laptops are not considered accessories and are out of scope for this working group.
 
-Since most of the existing tracking accessories use Bluetooth, the DULT WG will coordinate as needed with the IETF 6lo WG and Bluetooth SIG.
+The WG will work with gender-based violence experts. Since most of the existing tracking accessories use Bluetooth, the DULT WG will coordinate as needed with the IETF 6lo WG and Bluetooth SIG.
```
Comment (ekr) [inline charter/charter-detecting-unwanted-location-trackers.md:52]: This feels like a funny juxtaposition. The rest of the text in this graf is very clear about what we are going to do in that the BT SIG is a concrete thing. But like "GBV experts" is not a clearly specified group

I would argue that the text in PR#111 is sufficient, or maybe "will consult with GBV specialists throughout the process and vet with them prior to publication"

Comment (npdoty) [reply charter/charter-detecting-unwanted-location-trackers.md:52]: 111 suggests that gender-based violence experts should just be consulted at the end of the process before publication to confirm efficacy. That's specifically contrary to the agreement in the meeting that we want to work with these advocates and experts throughout the standardization process.

Comment (maggiedelano) [reply charter/charter-detecting-unwanted-location-trackers.md:52]: That looks better to me. I think this is more in the spirit of what we discussed at the BoF.

**Comment by tfpauly** [comment]
@bledvina can this be closed in favor of #111?

**Pull request closed without merging**